content/applications/hr/employees/new_employee.rst

```diff
@@ -57,22 +58,22 @@ Optional fields
         :align: center
         :alt: Both job position fields entered but with different information.
 
-- Tags: Click on a tag in the drop-down menu to add any tags applicable to the employee. Any tag can
+- Tags: click on a tag in the drop-down menu to add any tags applicable to the employee. Any tag can
```

Comment:
I am adding it because it looks better with al of them guilabeled.  My logic for not using a gui was that the word isn't bolded, and it disappears after you select a tag. The name is more of a placeholder and disappears after yo assign the tags. But I am changing it to a gui, which looks better overall.